src/App.css

```diff
@@ -1,39 +1,12 @@
-.App {
-  text-align: center;
-}
-
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
-}
-
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
-}
-
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
+body {
   display: flex;
-  flex-direction: column;
-  align-items: center;
   justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
-}
-
-.App-link {
-  color: #61dafb;
+  align-items: center;
 }
 
-@keyframes App-logo-spin {
-  from {
-    transform: rotate(0deg);
-  }
-
-  to {
-    transform: rotate(360deg);
-  }
+.App {
```

Comment:
- [ ] Somehow, your app does not match the given wirefram. Please fix it as well :)


Your image            |  Original image
:-------------------------:|:-------------------------:
<img width="1440" alt="Screenshot 2023-01-03 at 18 29 21" src="https://github.com/ahmed025566/Math-Magicians/assets/69011963/060d5044-df13-4be9-ab51-0600e644a1e7"> | <img src="https://github.com/ahmed025566/Math-Magicians/assets/69011963/a154f53a-2d81-445f-85e7-05ecfee1b769" width="1440"/>